### DISCUSSION TITLE
Better skin compatibility & issues with layout editor.

### DISCUSSION THREAD

**Discussion by h4rldev** [ppy/osu#24265] [Ideas]
### The Issue(s)
- A lot of skins break when imported into osu!lazer, causing some things to change, like combo counter breaking, pieces of the column breaking (on mania). 
- Fixing a lot of these changes seem to be more problematic since you can't seem change the skin by file any longer (or as far as I know).
- The skin layout editor doesn't properly preview position on the screen, causing misalignment when getting out of the layout editor.

### Solution
- Implement a way for lazer to keep osu!stable skins from breaking by retaining their original positions and such.
- Probably include proper skin.ini support (If it doesn't exist of course), allowing the skin to be edited, and the folder previewed with a button in the settings, like in stable.

**Comment by peppy** [comment]
Please include your skin as a link or attachment and show what is broken so we can fix it.

**Reply by h4rldev** [reply]
I've seen someone trying to/asking to implement the old combo counter into lazer, will that be linked to this?
Also, let me quickly export the osk, I just need to open up stable real quick.

**Reply by h4rldev** [reply]
https://cdn.discordapp.com/attachments/936760442628472882/1130340553406554142/pl0xchordxx_Ezze_2021_edit.osk

**Reply by peppy** [reply]
I'm not sure what "old combo counter" means. Please provide specifics (screenshots pointing to what you expect and what you're getting).

**Reply by h4rldev** [reply]
### Should I say LegacyComboCounter?
- lazer 
![image](https://github.com/ppy/osu/assets/98224660/8241ae6e-848f-4a1d-b198-74f25219c32a)

- stable for reference

![image](https://github.com/ppy/osu/assets/98224660/3ca147ac-dbc7-4a91-8caf-87556c8accbe)
